mycroft_bus_client/message.py

```diff
@@ -142,3 +161,32 @@ def publish(self, msg_type, data, context=None):
             del new_context['target']
 
         return Message(msg_type, data, context=new_context)
+
+    def utterance_remainder(self):
+        """
+        For intents get the portion not consumed by Adapt.
+
+        For example: if they say 'Turn on the family room light' and there are
+        entity matches for "turn on" and "light", then it will leave behind
+        " the family room " which is then normalized to "family room".
+
+        Returns:
+            str: Leftover words or None if not an utterance.
+        """
+        utt = normalize(self.data.get("utterance", ""))
```

Comment:
should lingua_franca be dragged into this package?

Reply:
in fact, maybe this shouldnt be here at all or needs a big refactor
- now normalized and non normalized utterances are passed to adapt
- padatious does not work with this

i'd say we just drop the normalize step for now

Reply:
A true, missed that this method got added as well. I think the entire "utterance_remainder" method should be dropped since that is very intent handler specific.

Reply:
Makes sense to me; I've never used this method. I'll remove it.